cross/armcc.txt

```diff
@@ -0,0 +1,11 @@
+[binaries]
+c = 'armcc'
+cpp = 'armcc'
+ar = 'armar'
+strip = 'armar'
+
+[host_machine]
+system = 'WINDOWS'
+cpu_family = 'arm'
+cpu = 'Cortex-M0+'
```

Comment:
Again, the convention is to have these in lower case (in Meson tree, people can set these to anything they want in their own projects).

We have also by convention had `cpu` as the exact processor string that the operating system reports. Something like `armv7hlv`. I'm not familiar enough with this use case to say if this is a good policy to follow in this particular case. You can choose what to have here, just throwing this information out here to help you make the call.